adal/tests/Test.ADAL.NET.Unit/Test.ADAL.NET.Unit.csproj

```diff
@@ -58,6 +58,7 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.7.2" />
     <PackageReference Include="MSTest.TestAdapter" Version="1.3.1" />
     <PackageReference Include="MSTest.TestFramework" Version="1.3.1" />
+    <PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="5.1.5" />
```

Comment:
><PackageReference Include="System.IdentityModel.Tokens.Jwt" Version="5.1.5" /> [](start = 4, length = 78)

remember to register the usage. Also I believe the latest version is 5.2 ?

Reply:
#Resolved

Reply:
I do not see this updated to the latest version

---
In reply to: [207337639](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1124#discussion_r207337639) [](ancestors = 207337639)

Reply:
? still not updated to the latest version

---
In reply to: [207600270](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1124#discussion_r207600270) [](ancestors = 207600270,207337639)

Reply:
Please update to latest System.IdentityModel ...

---
In reply to: [208421892](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1124#discussion_r208421892) [](ancestors = 208421892,207600270,207337639)

Reply:
#Resolved